### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_Keyboard:Keyboard focus does not go on "Add Tag" button provided on the page until manually click action is performed on the textbox present in the cell i.e.Textbox present besides "Add Tag" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4197]
“Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
Environment Details:
Application Name: Vienna Jupyter Notebook Widgets
URL: https://notebooks.azure.com/ahcan/libraries/amlsdkwidgets
Browser Details: Microsoft Edge 42.17134.1.0
Operating System: Windows10

Repro Steps
1)Launch the URL:https://notebooks.azure.com/ahcan/libraries/amlsdkwidgets
2)Run the widgets.ipynb notebook
3)Navigate to the cells present on the page-->Navigate to "Add Tag"

Actual
Keyboard focus does not go on "Add Tag" button provided on the page until manually click action is performed on the textbox present in the cell i.e.Textbox present besides "Add Tag" button.

Common scenario is observed for all the control such as links and buttons present on the page.

Expected
Keyboard focus should go on "Add Tag" button provided on the page.

User Impact
People who are blind (who cannot use devices such as mice that require eye-hand coordination)
People with low vision (who may have trouble finding or tracking a pointer indicator on screen)
Some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard 

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}}

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
